src/constants/SeatsData.ts

```diff
@@ -0,0 +1,85 @@
+export const SEATS_DATA_VIP = [
```

Comment:
the file of constants should start with the small letter
seatsData.ts
because it's not a component

Reply:
done